chart/estafette-k8s-hpa-scaler/templates/_helpers.tpl

```diff
@@ -0,0 +1,14 @@
+{{- define "commonLabels" -}}
+app: {{ .Chart.Name | quote }}
+release: {{ .Release.Name }}
+chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+heritage: {{ .Release.Service }}
+{{- if .Values.extraLabels }}
+{{ toYaml .Values.extraLabels }}
+{{- end }}
+{{- end -}}
+
+{{- define "completeLabels" -}}
+{{ include "commonLabels" . }}
+appVersion: {{.Chart.AppVersion | quote}}
```

Comment:
On all current deployments we always have a `version` label which is shown in a lot of grafana dashboards. We need to make sure that at least in our own deployments it's added, regardless of whether we add it to this public helm chart.

Reply:
I see. I believe we can rename this label to `version` actually. Because the chart version is already covered in the `chart` label.